src/Tizen.NUI/src/internal/Interop/Interop.QuickPanelClient.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Tizen.NUI
+{
+    internal static partial class Interop
+    {
+        internal static partial class QuickPanelClient
+        {
+            const string lib = "libtzsh_quickpanel.so.0";
+
+            [global::System.Runtime.InteropServices.DllImport(lib, EntryPoint = "tzsh_quickpanel_create_with_type")]
+            internal static extern IntPtr tzshQuickPanelCreateWithType(IntPtr tzsh, IntPtr win, int type);
```

Comment:
Change names of all P/Invoke methods to follow the naming convention.
You can choose 2 options:
1) follow .net framework naming guide for method : https://docs.microsoft.com/ko-kr/dotnet/standard/design-guidelines/names-of-type-members#names-of-methods
    for example: CreateWithType()
or,
2) use name of native's one
```suggestion
            [global::System.Runtime.InteropServices.DllImport(lib)]
            internal static extern IntPtr tzsh_quickpanel_create_with_type(IntPtr tzsh, IntPtr win, int type);
```

Reply:
I'll fix it follow 1st option.